mirobo/airpurifier.py

```diff
@@ -0,0 +1,63 @@
+from .device import Device
+from .containers import AirPurifierStatus
+
+
+class AirPurifier(Device):
+    """Main class representing the air purifier."""
+
+    def status(self):
+        """Retrieve properties."""
+
+        # A few more properties:
+        properties = ['power', 'aqi', 'humidity', 'temp_dec',
+                      'mode', 'led', 'led_b', 'buzzer', 'child_lock',
+                      'limit_hum', 'trans_level', 'bright',
+                      'favorite_level', 'filter1_life', 'act_det',
+                      'f1_hour_used', 'use_time', 'motor1_speed']
+
+        values = self.send(
+            "get_prop",
+            properties
+        )
```

Comment:
Considering that the get_prop is pretty generic (it's also used for the strip it seems, as well as for the yeelights), maybe it makes sense to have a list (or a mapping?) of properties and their corresponding types listed and move the status() implementation to the parent class at some point? See also the comments below regarding to property names.